beacon_node/lighthouse_network/src/discovery/mod.rs

```diff
@@ -550,8 +550,6 @@ impl<TSpec: EthSpec> Discovery<TSpec> {
         if let Ok(node_id) = peer_id_to_node_id(peer_id) {
             // If we could convert this peer id, remove it from the DHT and ban it from discovery.
             self.discv5.ban_node(&node_id, None);
-            // Remove the node from the routing table.
-            self.discv5.remove_node(&node_id);
```

Comment:
I think we still want to remove bad nodes from our discovery routing table. This will prevent us from advertising them to other peers

Reply:
yeah, the reason I removed is because `ban_node` already `removes_node`:
https://github.com/sigp/discv5/blob/1439decd4e7d7c9de78ef61b5d67be3fee688510/src/discv5.rs#L347-L349

Reply:
Oh nice! Didn't notice that!